pkg/virt-controller/watch/application.go

```diff
@@ -311,10 +322,12 @@ func Execute() {
 	app.kubeVirtInformer = app.informerFactory.KubeVirt()
 	app.informerFactory.Start(stopChan)
 
-	app.kubeVirtInformer.SetWatchErrorHandler(func(r *cache.Reflector, err error) {
+	if err := app.kubeVirtInformer.SetWatchErrorHandler(func(r *cache.Reflector, err error) {
 		apiHealthVersion.Clear()
 		cache.DefaultWatchErrorHandler(r, err)
-	})
+	}); err != nil {
+		log.Log.Warningf("failed to set the watch error handler: %v", err)
```

Comment:
Do you think it would make sense to fail on error here? Like the other errors in this file.

Reply:
Does `golog.Fatalf` make sense to you ? (I've seen it being used in this file)